sqlitecluster/SQLite.h

```diff
@@ -206,6 +215,9 @@ class SQLite {
         // This is the last committed hash by *any* thread for this file.
         atomic<string> _lastCommittedHash;
 
+        // An identifier used to choose the next journal table to use with this set of DB handles.
+        atomic<int64_t> _nextJournalCount;
```

Comment:
No, when we create new DB handles, we want to hand out journal tables to them in order.

So that when we do:

```
SQLite db1(constructor, args, here);
SQLite db2(constructor, args, here);
SQLite db3(constructor, args, here);
```

These three objects each use a different journal. We use this counter to hand out a new number each time the constructor is called, and a journal table will be assigned based on this value modulo `_journalNames.size()`, but we still need to keep an offset somewhere.